src/FixedSizeArrays.jl

```diff
@@ -189,7 +189,7 @@ end
 
 # unsafe: the native address of the array's storage
 
-Base.unsafe_convert(::Type{Ptr{T}}, a::FixedSizeArray{T}) where {T} = Base.unsafe_convert(Ptr{T}, a.mem)
+Base.cconvert(::Type{<:Ptr}, a::FixedSizeArray) = a.mem
```

Comment:
Wait, what happens if you try to do `cconvert(Ptr{UInt8}, FixedSizeVector{Float64})`?

Reply:
The `cconvert` method for `Array` is permissive in the same manner:

https://github.com/JuliaLang/julia/blob/57b9b591e28181aee8ff985d0ace661b408577d7/base/pointer.jl#L65